libafl_targets/build.rs

```diff
@@ -156,8 +156,13 @@ fn main() {
         .file(src_dir.join("cmplog.c"))
         .compile("cmplog");
 
-    #[cfg(any(target_os = "linux", target_os = "freebsd"))]
+    #[cfg(feature = "builtin_forkserver")]
     {
+        #[cfg(not(any(target_os = "linux", target_os = "freebsd")))]
+        {
+            compile_error!("Incompatibility detected; we cannot build the builtin forkserver for anything but Linux and FreeBSD targets. Did you mean to use the `forkserver` feature?");
```

Comment:
Why wouldn't it work on MacOS?

Reply:
Ahhh wait, I am dumb and misunderstood the issue. One moment.